politeiawww/cmsnotifications.go

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"text/template"
+	"time"
+
+	"github.com/decred/politeia/politeiawww/user"
+)
+
+var (
+	templateInvoiceNotification = template.Must(
```

Comment:
All of the other email template declarations are in either `politeiawww.go` or `userwww.go`, depending on what route they're used in.  Can we move this to `cmswww.go` to continue this pattern.